chips/README.md

```diff
@@ -0,0 +1,294 @@
+An icon or an image can be used to give information about the kind of information.
+
+#### Text alone 
+
+```kotlin
+object VitaminChips {
+
+    @Composable
+    fun Input(
+        label: String,
+        modifier: Modifier = Modifier,
+        enabled: Boolean = true,
+        colors: ChipColors = VitaminChipColors.default(),
+        sizes: ChipSizes = VitaminChipSizes.medium(),
+        ripple: RippleTheme = VitaminChipRipples.default(),
+        onClick: () -> Unit
+    )
+}
+```
+
+Parameters | Descriptions
+-- | --
+`label: String` | Text to be displayed into the Chip
+`selected: Boolean,` | True if the Chip is selected, otherwise false
+`modifier: Modifier = Modifier` | `Modifier` to be applied to the component
+`enabled: Boolean = true` | True if you can click on the Chip, otherwise false
+`colors: ChipColors = VitaminChipColors.default()` | Colors to be applied to the Chip
+`sizes: ChipSizes = VitaminChipSizes.medium()` | Sizes to be applied to the Chip. (VitaminChipSizes.medium() | VitaminChipSizes.small())
+`ripple: RippleTheme` | The ripple effect to be applied to the Chip
+`onClick: (() -> Unit)?` | Callback to be called when the Chip is clicked
+
+_In your project :_
+```kotlin
+VitaminChips.Input(
+    label = "Walking",
+    onClick = {
+        // Callback to handle Chip state 
+    }
+)
+```
+
+#### With Icon
+
+```kotlin
+object VitaminChips {
+
+    @Composable
+    fun InputWithIcon(
+        label: String,
+        modifier: Modifier = Modifier,
+        enabled: Boolean = true,
+        colors: ChipColors = VitaminChipColors.default(),
+        sizes: ChipSizes = VitaminChipSizes.medium(),
+        ripple: RippleTheme = VitaminChipRipples.default(),
+        onClick: () -> Unit,
+        leadingIcon: (@Composable () -> Unit)
+    )
+}
+```
+
+Parameters | Descriptions
+-- | --
+`label: String` | Text to be displayed into the Chip
+`selected: Boolean,` | True if the Chip is selected, otherwise false
+`modifier: Modifier = Modifier` | `Modifier` to be applied to the component
+`enabled: Boolean = true` | True if you can click on the Chip, otherwise false
+`colors: ChipColors = VitaminChipColors.default()` | Colors to be applied to the Chip
+`sizes: ChipSizes = VitaminChipSizes.medium()` | Sizes to be applied to the Chip. (VitaminChipSizes.medium() | VitaminChipSizes.small())
+`ripple: RippleTheme` | The ripple effect to be applied to the Chip
+`onClick: (() -> Unit)?` | Callback to be called when the Chip is clicked
+`leadingIcon: (@Composable () -> Unit)` | Icon Content to be displayed before label into the Chip
+
+_In your project :_
+```kotlin
+VitaminChips.InputWithIcon(
+    label = "Walking",
+    onClick = {
+        // Callback to handle Chip state 
+    },
+    leadingIcon = {
+        Icon(
+            painter = painterResource(
+                id = R.drawable.ic_vtmn_walk_line
+            ),
+            contentDescription = null
+        )
+    }
+)
+```
+
+#### With Image
```

Comment:
I'm not sure to understand the difference between `WithImage` and `WithIcon` 🤔 
And after checked the implementation, it seems that it is the exact same implementation

Reply:
Kept only one variant for input with optional leadingIcon and leadingContent